zeek-mode.el

```diff
@@ -147,6 +147,12 @@ reflected in the parse tree."
     (interactive)
     (zeek-command-on-buffer (zeek-script-cmd "parse" "-") "*zeek-script parse tree*"))
 
+  (defun zeek-format-before-save ()
+    "Add this to .emacs to run zeek-format on the current buffer when saving:
+\(add-hook 'before-save-hook #'zeek-format-before-save)"
```

Comment:
Yacin do we need that `\` in there or is that an editing leftover?

Reply:
It doesn't look like it. I saw it in [go-mode](https://github.com/dominikh/go-mode.el/blob/master/go-mode.el#L1777) whenever an s-expression is referenced in a functions docstring, so I assumed it was needed. Digging into it more, I found [in the manual](https://www.gnu.org/software/emacs/manual/html_node/elisp/Function-Documentation.html) that this has some benefit but I don't quite understand it myself. I guess blindly copying folks sometimes pays off!

Reply:
Interesting! I hadn't come across this. It looks fine when I say `C-h f zeek-format-before-save`:

```
zeek-format-before-save is an interactive Lisp function in ‘zeek-mode.el’.

(zeek-format-before-save)

Add this to .emacs to run zeek-format on the current buffer when saving:
(add-hook ’before-save-hook #’zeek-format-before-save)
```

Let's leave it in. :)